File: .NET-app/MorkosiaPrepaLeague/Models/Core/ActivityType.cs
namespace MorkosiaPrepaLeague.Models.Core
{
    public class MplActivityType : CosmosDocumentBase
    {
        public string Name { get; set; } = string.Empty;

        public int DefaultPoints { get; set; }

        public string LeagueId { get; set; } = string.Empty;

        public List<string> LinkedPositions { get; set; } = new();
    }
}

File: src/app/models/activity_type.py
from typing import List

from beanie import Indexed

from app.models.base import DocumentBase


class ActivityType(DocumentBase):
    name: str = ""
    default_points: int = 0
    league_id: Indexed(str) = ""  # type: ignore[valid-type]
    linked_positions: List[str] = []

    class Settings:
        name = "activity_types"
